src/saturn_engine/worker/executors/queue.py

```diff
@@ -53,21 +101,28 @@ async def run_queue(self) -> None:
                 async with processable._context:
 
                     @self.services.s.hooks.message_executed.emit
-                    async def scope(xmsg: ExecutableMessage) -> PipelineResults:
-                        return await self.executor.process_message(xmsg.message)
-
+                    async def scope(
+                        xmsg: ExecutableMessage,
+                    ) -> Optional[PipelineResults]:
+                        try:
+                            return await self.executor.process_message(xmsg.message)
+                        except Exception as err:
+                            await self._process_error(xmsg, err)
+
+                    output = None
                     try:
                         output = await scope(processable)
-                    except Exception:  # noqa: S110
+                    except:
```

Comment:
Always keep at least `except Exception`, otherwise it's gonna catch anything which gets very annoying (block signal or `ExitException`.